README.md

```diff
@@ -1,20 +1,20 @@
-# WooCommerce MyParcel
-Welcome to the WooCommerce MyParcel repository on GitHub. Here you can browse the source, look at open issues and keep track of development.
+# WooCommerce PostNL
+Welcome to the WooCommerce PostNL repository on GitHub. Here you can browse the source, look at open issues and keep track of development.
 
-This WooCommerce extension allows you to export your orders to the MyParcel service (www.myparcel.nl). Single orders exports aswell as batch exports are possible.
+This WooCommerce extension allows you to export your orders to the PostNL service (www.postnl.nl). Single orders exports aswell as batch exports are possible.
 
 ## Main features
 - Delivery options integrated in your checkout
-- Export your WooCommerce orders to MyParcel with a simple click, single orders or in batch
+- Export your WooCommerce orders to PostNL with a simple click, single orders or in batch
 - Print shipping labels directly (PDF)
 - Create multiple shipments for the same order
 - Choose your package type (Parcel, mailbox package or unpaid letter)
-- Define preset MyParcel shipping options (signature required, extra insurance, etc.)
-- Modify the MyParcel shipping options per order before exporting
+- Define preset PostNL shipping options (signature required, extra insurance, etc.)
+- Modify the PostNL shipping options per order before exporting
 - Extra checkout fields to separate street name, house number and house number suffix for more precise address data
 - View the status of the shipment in the order details page
 - Add track&trace link to the order confirmation email
 
-A MyParcel API account is required for this plugin! You can create this in your account or contact MyParcel at info@myparcel.nl
+A PostNL API account is required for this plugin! You can create this in your account or contact PostNL at info@postnl.nl
```

Comment:
Maak maar een trello kaart aan om deze zin aan te passen als duidelijk is wat het e-mail adres moet zijn